### PR TITLE
Remove IE8 support from Origin

### PR DESCRIPTION
**Pull request by paddyduke**
This branch simply reverts the changes made here: https://github.com/fac/origin/pull/67

Removes all the IE 8 fallbacks from Origin now that we no longer support any versions below IE 11.

### CODE REVIEW

**Comment by craigcwaveformlabs** [comment]
PASS +1 Looks good to me as a reversion removing the temporary changes.

File: assets/scss/local/fonts/_font-settings.scss [hunk 1]
```diff
@@ -6,14 +6,6 @@
 $font-path: "../fonts" !default;
 
 /**
- * If `false`, references to EOT files won’t be rendered. EOT files are only
- * required for `@font-face` to work in < IE9, or IE9 in Compatbility Mode.
- * In time we’ll remove this rule as we drop support for IE8.
- */
-
-$support-ie8: true !default;
```
Comment (Lindisaurus) [inline assets/scss/local/fonts/_font-settings.scss]: Worth deleting the EOT files from Origin too? https://github.com/fac/origin/tree/master/assets/fonts/SourceSansPro

Comment (paddyduke) [inline assets/scss/local/fonts/_font-settings.scss]: Oops, yep!

Comment (paddyduke) [inline assets/scss/local/fonts/_font-settings.scss]: Wait, I already did.

screen shot 2017-01-24 at 12 36 48

Comment (Lindisaurus) [inline assets/scss/local/fonts/_font-settings.scss]: :man_facepalming:

**Pull request merged**